packages/contracts/scripts/testrpc.ts

```diff
@@ -0,0 +1,44 @@
+import * as testrpc from 'ganache-cli';
```

Comment:
Ok. Got it. We want control over private keys in accounts. But why can't we use our standard mnemonic and just get the private keys out of it? Maybe I still don't understand that approach, but it still seems hacky.

Reply:
And I'm more of a fan of using in-process testrpc. That's faster. Look at sol-cover PR. After it gets merged - we'll have it in web3Factory.